WatchApp/LoopWatchApp.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  LoopWatchApp.swift
+//  Loop
+//
+//  Created by Pete Schwamb on 9/21/25.
+//  Copyright © 2025 LoopKit Authors. All rights reserved.
+//
+import SwiftUI
+
+@main
+struct LoopWatchApp: App {
+    @WKApplicationDelegateAdaptor(ExtensionDelegate.self) private var appDelegate
+
+    var loopManager = LoopDataManager.shared
+
+    var body: some Scene {
+        WindowGroup {
+            ContentView()
+                .environment(loopManager)
+        }
+    }
+}
```

Comment:
nit: There's a new-ish way to register new environments a little more cleanly [using the @Entry macro](https://www.google.com/url?sa=t&source=web&rct=j&opi=89978449&url=https://developer.apple.com/documentation/swiftui/entry()&ved=2ahUKEwink5Cpwu-PAxXtDTQIHT3ZApkQFnoECBcQAQ&usg=AOvVaw00aUvoauvyyQcgx4JL2PKx)

```suggestion
@main
struct LoopWatchApp: App {
    @WKApplicationDelegateAdaptor(ExtensionDelegate.self) private var appDelegate
    
    @Environment(\.loopManager) var loopManager // not needed here but an example of usage

    var body: some Scene {
        WindowGroup {
            ContentView()
        }
    }
}

extension EnvironmentValues {
    @Entry var loopManager = LoopDataManager.shared
}
```

Reply:
Interesting.  LoopDataManager.shared is @MainActor, so it can't be accessed from a non-isolated context like that. I keep hitting that constraint; seems like there should be a special exception for stuff like this.